client/mm/mm_simple_arb.go

```diff
@@ -465,6 +465,7 @@ func (a *simpleArbMarketMaker) botLoop(ctx context.Context) (*sync.WaitGroup, er
 					a.rebalance(epoch.Current)
 				}
 			case <-ctx.Done():
+				bookFeed.Close()
```

Comment:
Looks like you need to close this if you get an error from `SubscribeMarket` above too.